src/schema/schema.graphql

```diff
@@ -3423,18 +3378,18 @@ type Mutation {
 	revokeRights(input: RevokeRightsInput!): RightsResponse
 	revokeTrustedDevice: Boolean
 	saveDraft(message: SendMessageInput!, accountName: String): SaveDraftResponse
-	sendMessage(
-		message: SendMessageInput!
+	sendMessage(message: SendMessageInput!, accountName: String): SendMessageResponse
+	sendSecureMessage(
+		message: SendSecureMessageInput!
 		accountName: String
+		sign: Boolean
```

Comment:
these two fields are different here for sendSecureMessage, can we update sendMessage mutation to have these optional arguments? so we don't have to introduce new mutation/apis ?

Reply:
Fixed.